src/Service/Program.cs

```diff
@@ -23,6 +24,13 @@ public class Program
 
         public static void Main(string[] args)
         {
+            if (!ValidateAspNetCoreUrls())
+            {
+                Console.Error.WriteLine("Invalid ASPNETCORE_URLS format. e.g.: ASPNETCORE_URLS=\"http://localhost:5000;https://localhost:5001\"");
+                Environment.ExitCode = -1;
```

Comment:
Can we use an enumeration instead of simply an integer, just to communicate what the code means more clearly?

Reply:
actually I see now that we were aclready just using -1 to mean failure generally, so probably don't need an enum here, since this isn't specific to the kind of failure you've added.